src/web/tests/jest.setup.js

```diff
@@ -1,19 +1,19 @@
 // Provide the Web API ResizeObserver interface to Jest
 global.ResizeObserver = class ResizeObserver {
     constructor(callback) {
-      this.callback = callback;
+        this.callback = callback;
     }
-  
+
     observe(target) {
-      this.callback([{ target }]);
+        this.callback([{ target }]);
     }
-  
+
     unobserve() {}
-  
+
     disconnect() {}
 };
 
 // Silence output while running tests
-global.console.log = jest.fn()
-global.console.error = jest.fn()
-global.console.debug = jest.fn()
+global.console.log = jest.fn();
+global.console.error = jest.fn();
```

Comment:
I'm wondering if we should let errors show up.

Reply:
If we let errors show through, then we can use "errors" during debug, too.  Which is nice.